### ISSUE TITLE
Add more workers to run models in parallel

### ISSUE THREAD

**Issue by iamianM** [oobabooga/text-generation-webui#2585]
**Description**

Add an argument --workers, which specifies how many workers will run, similar to uvicorn. This would allow for multiple instances of the model to run in threads such that multiple requests could be run concurrently.

**Comment by Nixellion** [comment]
Would it not multiply VRAM requirements for each worker?

**Comment by iamianM** [comment]
Yeah I'm on a A100 and want to host 3 models, but load balancing between the 3 endpoints is quite annoying. Would be much more beneficial to be able to add workers to the one endpoint

**Comment by tallesairan** [comment]
Does anyone know if it is possible to use gptq models concurrently? I really need to request multiple times at the same time, the api has a request lock it seems
